src/main/java/bio/terra/stairway/Database.java

```diff
@@ -203,17 +198,17 @@ private void create() {
                             "(flightid VARCHAR(36) PRIMARY KEY," +
                             " submit_time TIMESTAMP NOT NULL," +
                             " class_name TEXT NOT NULL," +
-                            " input_parameters JSONB," +
+                            " input_parameters TEXT," +
                             " completed_time TIMESTAMP," +
-                            " output_parameters JSONB," +
+                            " output_parameters TEXT," +
                             " succeeded BOOLEAN," +
                             " error_message TEXT)");
 
             statement.executeUpdate(
                     "CREATE TABLE " + FLIGHT_LOG_TABLE +
                             "(flightid VARCHAR(36)," +
                             " log_time TIMESTAMP NOT NULL," +
-                            " working_parameters JSONB NOT NULL," +
+                            " working_parameters TEXT NOT NULL," +
```

Comment:
What caused the switch here? Is there a way to use JSONB with preparedStatements? I remember in psycopg2 that I would need to wrap a JSON string in a JSON Adapter when preparing the argument to go into a JSONB column -- wondering if there is something similar we can use here.

Reply:
We are not using the capabilities of JSONB, so rather than fuss with SQL interface, it is simpler to put it into TEXT. Since we always run JSON through Jackson on the way in and out, I am not concerned about storing junk.